src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -3657,10 +3667,11 @@ public void EscapeSurrogatePairsThrowsHighSurrogateMissing()
             "{" + ValidUtf8 + ":" + ValidUtf8 + "}")]
         public void UTF8ReplacementCharacters(bool skipValidation, string expected0, string expected1, string expected2, string expected3)
         {
+            // SkipValidation does not affect whether we write the replacement character or not (we always do, unless we add a new option to control).
```

Comment:
I wonder if this is worth documenting, maybe as a remark somewhere relevant. Seems too much of an edge case to do so on the option itself, but maybe it's good enough. Otherwise, maybe we add this remark to each of the `WriteXx` overloads that take strings.

Reply:
Hmm the S.T.E.W API doc for Encode does not currently document this behavior.

Have you heard any asks for this functionality in the future and\or complaints about the previous behavior (which would throw instead of using the replacement char (like [this API](https://github.com/dotnet/corefx/blob/c5932104a914e4ec95a14a867da8bd109781e4e2/src/Common/src/CoreLib/System/Text/Unicode/Utf8.cs#L42) has via `replaceInvalidSequences` parameter)?

Reply:
> Hmm the S.T.E.W API doc for Encode does not currently document this behavior.

The SkipValidation concept is specific to the JSON APIs, not S.T.E.W. It is conceivable that someone would think that skipvalidation applies to replacement character vs leaving things as is, but I don't have any evidence of this. We wouldn't write invalid utf-8 as is anyway (we would either throw or replace). But your point is well taken, especially since I doubt anyone would care. No action needed.